CONTRIBUTING.md

```diff
@@ -0,0 +1,27 @@
+# How to Contribute
+
+Contributing to this project entails submitting pull requests (PRs) to this repository. PRs need to be appropriately assigned and labelled.
+
+1. Fork this repository.
+2. Clone your fork of the repository.
+3. When developing, develop and commit in a new, local branch. Leave the protected branches (e.g. develop) alone. They are needed to fetch from upstream.
+   1. If your changes are comprised of multiple commits, use `git rebase -i` to reduce the changes to a single commit. Otherwise, use `git amend` to add files to a commit.
+4. When you are ready to make a PR, fetch from upstream to make sure you have the latest changes from the upstream repository. Add your commit on top of any new commits: `git cherrypick <commit hash>`
+5. Set your local branch to track its equivalent on the remote server. Push your changes to that remote branch.
+6. Create a draft PR.
+7. Add the appropriate label to the draft. If you do not have the ability to do so, reach out to Marcin Wadon (marcin.wadon@gmail.com).
+   - **feature** - for all the things that are kinda “new” or introduce a significant change to the codebase. It also means that the minor part of the version number of the release will increment.
+   - **fix** - it’s for fixes to the existing codebase (fixes to features, not the tools). This will increment the patch part of the version number, if merged to hotfix branch, or the minor part of the version number, if merged to develop along other development changes.
+   - **test** - it’s for PRs containing just tests or test tools.
+   - **adr** - it's for PRs related to architecture decision records.
+   - **other** - it’s for everything else.
+8. Assign someone on the protocol team (e.g., marcineadon) to reveiw your PR.
+9. Make sure that the tests run on the PR finish successfully.
+10. Mark the PR as _ready for review_.
+
+# Running SBT in the command-line
+
+Run these commands from the base directory of this project.
+
+- `sbt compile` - compiles the project.
+- `sbt tests` - runs the tests.
```

Comment:
`sbt test`

Reply:
I'd add one more thing here, `sbt runLinter` which auto-formats the code according to style guide. This isn't documented.

Reply:
Fixed typo: https://github.com/Constellation-Labs/tessellation/pull/266/commits/228868cde7b40cfd716dd767e3f1a7bb1374f048

Added `sbt runLinter`: https://github.com/Constellation-Labs/tessellation/pull/266/commits/0ac5619c761465906296ab416cc941b1e26bf39a